README.md

```diff
@@ -72,7 +72,7 @@ $obj = $(Get-WMIObject Win32_UserAccount -Filter "Name='Guest'")
 $obj.Rename("OtherGuest")
 ~~~
 
-This has the added benefit of not requiring escaping '$' in the PowerShell code.
+This has the added benefit of not requiring escaping '$' in the PowerShell code. Note that templates need to have DOS linefeeds or they will not work as expected. One tool for converting UNIX linefeeds to DOS linefeeds is [unix2dos](http://freecode.com/projects/dos2unix).
```

Comment:
Yeah, I was not really suggesting that. The really nasty part in this is that no warning/error is printed if the template has linefeeds of the wrong kind. This makes debugging these problems really hard until one has experienced this once the hard way.

Regarding calling Posh... is it necessary to use cmd.exe as the middleman between Puppet and Powershell?